src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,17 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int caterpillarWidth;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(int caterpillarWidth, String name, String color) {
+        this.caterpillarWidth = caterpillarWidth;
+        this.setName(name);
+        this.setColor(color);
+    }
```

Comment:
```suggestion
    public Bulldozer(int caterpillarWidth, String name, String color) {
        this.caterpillarWidth = caterpillarWidth;
        setName(name);
        setColor(color);
    }
```

do we need this keywords here?